### DISCUSSION TITLE
How to create tooltip for DataGrid Column Header (not for Cells)

### DISCUSSION THREAD

**Discussion by MichailBarsky** [AvaloniaUI/Avalonia#13072] [Q&A] [answered]
Hello colleagues. Please tell me how to make a tooltip for a datagrid column header. I used a not very nice solution, like this

DataGridTextColumn Binding="{Binding Name}">
 <DataGridTextColumn.HeaderTemplate>
	< DataTemplate>
 	   < TextBlock  Text="Name" **ToolTip.Tip="Name of the person"** />
	< /DataTemplate>
  </DataGridTextColumn.HeaderTemplate>
</DataGridTextColumn>

But I understand what needs to be done somehow through:

< DataGrid.Styles>
	< Style Selector="DataGridColumnHeader">
		< Setter Property="ToolTip.Tip" **Value="???"** />
	</Style>
< /DataGrid.Styles>

Here I don’t understand how to bind a value to some DataGridColumn property

Any ideas ?

**Comment by timunie** [comment, marked as answer]
Can you try `HeaderTemplate` instead? 
You cannot really know the columns data. You could however bind to `self.Content` and use a Converter for example. `Content` should be the header text you defined iirc.

**Reply by MichailBarsky** [reply]
Thank you. I'll try
